chromadb/test/utils/cross_version.py

```diff
@@ -5,10 +5,15 @@
 from types import ModuleType
 from typing import Dict, List
 
-base_install_dir = tempfile.gettempdir() + "/persistence_test_chromadb_versions"
+base_install_dir = (
```

Comment:
prior to this change, `base_install_dir` ended up being the same value for all workers spawned by pytest-xdist (which I would not expect), leading to the workers clobbering each other during installs